src/main/java/me/flashyreese/mods/sodiumextra/client/gui/SodiumExtraGameOptionPages.java

```diff
@@ -208,6 +212,22 @@ public static OptionPage particle() {
                         .build()
                 )
                 .build());
+        Map<String, List<Identifier>> otherParticles = Registry.PARTICLE_TYPE.getIds().stream()
+                .filter(identifier -> !identifier.getNamespace().equals("minecraft"))
+                .collect(Collectors.groupingBy(Identifier::getNamespace));
+        otherParticles.forEach((namespace, identifiers) -> groups.add(identifiers.stream().collect(
+                OptionGroup::createBuilder,
+                (builder, identifier) -> builder.add(
+                        OptionImpl.createBuilder(boolean.class, sodiumExtraOpts)
+                                .setName(Text.translatable(identifier.toTranslationKey("options.particles")))
+                                .setTooltip(Text.translatable(identifier.toTranslationKey("options.particles").concat(".tooltip")))
+                                .setControl(TickBoxControl::new)
+                                .setBinding((opts, val) -> opts.particleSettings.otherMap.put(identifier, val),
+                                        opts -> opts.particleSettings.otherMap.get(identifier))
```

Comment:
Due to how the configuration is implemented, I think the workaround fix for this is to dynamically set the value for them.
```suggestion
                                        opts ->  {
                                            if (!opts.particleSettings.otherMap.containsKey(identifier))
                                                opts.particleSettings.otherMap.put(identifier, true);

                                            return opts.particleSettings.otherMap.get(identifier);
                                        }
```